app/rifcs/rif_cs_generator.rb

```diff
@@ -98,6 +98,13 @@ def build_rif_cs
             end
           end
 
+          xml.relatedObject do
+            xml.key wrapper_object.creator_name
```

Comment:
As discussed previously we need to alter the existing 'Creator' rif-cs field and not add a new one. The original Creator field was on line 111 (now line 117 since you added this code). Please delete the new code you have written and replace xml.key wrapper_object.created_by with become xml.key wrapper_object.creator_name and move contributors to appear underneath creator in the code.